tobias/utils/motifs.py

```diff
@@ -228,6 +254,10 @@ def to_file(self, path, fmt="pfm"):
 		fmt : string
 			Format of motif file (pfm/jaspar/meme/transfac)
 		"""
+  
+		# add file ext if needed
+		if not path.endswith(f".{fmt}"):
+			path += f".{fmt}"

```

Comment:
This is well meaning, but should not happen. If I want to name my jaspar file "myjaspar.txt" it should be called that! ;) And not end up being "myjaspar.txt.jaspar". The reason is, that the output path signifies what the output path will be, and there is no way for anyone using the function to know what happens inside.